From 667408a58fdbe7e7720a74f2acd79db7ed2d4529 Mon Sep 17 00:00:00 2001
From: Angelo Naselli <anaselli@linux.it>
Date: Sun, 23 Mar 2014 21:52:45 +0100
Subject: [PATCH 2/9] Added YMGAAboutDialog

---
 swig/yui.i | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/swig/yui.i b/swig/yui.i
index 606e3a7..a7528b9 100644
--- a/swig/yui.i
+++ b/swig/yui.i
@@ -100,6 +100,7 @@ SWIGEXPORT void Init__yui(void) {
 #include "yui/YUILoader.h"
 
 #include "yui/mga/YMGA_CBTable.h"
+#include "yui/mga/YMGAAboutDialog.h"
 #include "yui/mga/YMGAWidgetExtensionFactory.h"
 
 using namespace std;
@@ -258,6 +259,7 @@ class Exception;
 %include yui/YExternalWidgetFactory.h
 %include yui/YExternalWidgets.h
 %include yui/mga/YMGA_CBTable.h
+%include yui/mga/YMGAAboutDialog.h
 %include yui/mga/YMGAWidgetExtensionFactory.h
 
 #if defined(SWIGRUBY)
-- 
2.18.0

