From 7b5939cf5b1e5a78dc3c7850f32db12014be8c9d Mon Sep 17 00:00:00 2001
From: Angelo Naselli <anaselli@linux.it>
Date: Thu, 19 Feb 2015 16:44:58 +0100
Subject: [PATCH 4/9] wrong merge :(

---
 swig/yui.i | 1 -
 1 file changed, 1 deletion(-)

diff --git a/swig/yui.i b/swig/yui.i
index f471e91..5dd5f82 100644
--- a/swig/yui.i
+++ b/swig/yui.i
@@ -175,7 +175,6 @@ class Exception;
 %apply SWIGTYPE *DISOWN { YTableHeader *header_disown };
 %apply SWIGTYPE *DISOWN { YWidget *parent_disown };
 
-<<<<<<< HEAD
 %include yui/YUILog.h
 %include yui/YUIPlugin.h
 
-- 
2.18.0

