From d7ec375fb44018daa4d9546e85d97a26d116a19b Mon Sep 17 00:00:00 2001
From: unknown <pasotti5@MIMIDR1C479.rete.poste>
Date: Wed, 1 Jul 2015 11:38:50 +0200
Subject: [PATCH 6/9] workaround for System.EntryPointNotFoundException (issue
 #18)

---
 swig/mono/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/swig/mono/CMakeLists.txt b/swig/mono/CMakeLists.txt
index 5cf0be4..6fefe5d 100644
--- a/swig/mono/CMakeLists.txt
+++ b/swig/mono/CMakeLists.txt
@@ -40,8 +40,8 @@ TARGET_LINK_LIBRARIES( yui_csharp ${LIBYUI_LIBRARY} )
 ADD_CUSTOM_COMMAND(
 	TARGET yui_csharp
 	POST_BUILD
-	COMMAND sed -i -e 's|DllImport\(\"yui\"|DllImport\(\"${MONO_LIBRARIES}/yui/yui.so\"|g' "${CMAKE_CURRENT_BINARY_DIR}/yuiPINVOKE.cs" )
-
+	COMMAND sed -i -e 's|DllImport\(\"yui\"|DllImport(\"$$\{MONO_LIBRARIES\}/yui/yui.so\"|g' "${CMAKE_CURRENT_BINARY_DIR}/yuiPINVOKE.cs" 
+)
 
 ADD_CUSTOM_COMMAND(
    TARGET yui_csharp POST_BUILD
-- 
2.18.0

